local-dev/Dockerfile

```diff
@@ -5,9 +5,10 @@ ENV PYTHONUNBUFFERED 1
 RUN export DEBIAN_FRONTEND=noninteractive
 RUN apt-get update \
   && apt-get -y install build-essential curl \
-  && apt-get -y install vim\ 
+  && apt-get -y install vim \ 
   && curl -sL https://deb.nodesource.com/setup_6.x | bash - \
-  && apt-get install -y nodejs \
+  && apt-get -y install nodejs \
```

Comment:
Due to someone's suggestion that npm's absence and our need to install it "manually" was due to nodejs installing improperly at this line, because typically an installation of nodejs brings npm with it. It was suggested I try testing if apt-get -y install nodejs, instead, made a difference (it did not seem to do so, but I left it like this for consistency of formatting)